### PR TITLE
Better user report when OMAPI error occurs or key is invalid

### PR DESCRIPTION
**Pull request by nerzhul**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
omapi_host

##### ANSIBLE VERSION
```
ansible 2.2.0.0
```

##### SUMMARY
Add better error handling for omapi_host on wrong user input

### CODE REVIEW

**Comment by nerzhul** [comment]
shipit

File: lib/ansible/modules/network/omapi_host.py [hunk 1]
```diff
@@ -147,6 +162,12 @@ def connect(self):
         try:
             self.omapi = Omapi(self.module.params['host'], self.module.params['port'], self.module.params['key_name'],
                                self.module.params['key'])
+        except binascii.Error:
+            self.module.fail_json(msg="Unable to open OMAPI connection. key_name is not a valid base64 key.")
```
Comment (gundalow) [inline lib/ansible/modules/network/omapi_host.py]: key_name -> key

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

**Comment by nerzhul** [comment]
fixed

**Review by gundalow — APPROVED** [review]

**Pull request merged**